src/main/java/org/jabref/gui/externalfiles/ImportHandler.java

```diff
@@ -116,7 +116,10 @@ protected List<ImportFilesResultItemViewModel> call() {
 
                     try {
                         if (FileUtil.isPDFFile(file)) {
-                            var pdfImporterResult = contentImporter.importPDFContent(file);
+                            var filePreferences = preferencesService.getFilePreferences();
+                            Path workingDir = bibDatabaseContext.getFirstExistingFileDir(filePreferences)
+                                    .orElse(filePreferences.getWorkingDirectory());
```

Comment:
JabRef's logic of storing files is more complicated.

JabRef searchers in EACH directory, not in ONE.

See org.jabref.logic.util.io.FileUtil#getListOfLinkedFiles.